transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/ButtonResponse.kt

```diff
@@ -15,3 +16,44 @@ data class ButtonResponse(
     val payload: String,
     val type: String,
 )
+/**
```

Comment:
Lets move Action,LinkAction and PostbackAction to separate file. Since they have nothing to do with ButtonResponse it will make more sense if they will be stored separately.